src/test/java/pages/CataloguePage.java

```diff
@@ -0,0 +1,97 @@
+package pages;
+
+import elements.Button;
+import elements.Text;
+import indices.Icons;
+import indices.Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.Type;
+import util.FindElementUtils;
+import util.FormatterUtils;
+import util.WaitUtils;
+
+
+public class CataloguePage extends BasePage {
+
+    @FindBy(xpath = "//h1[@class='catalog-navigation__title']")
+    protected Text catalogueName;
+
+    @FindBy(xpath = "//li[@data-id=%s]")
+    protected Button iconOptionBtn;
+
+    @FindBy(xpath = "//div[@data-id=%s]/descendant::div[%s]")
+    protected Button leftDropdownBtn;
+
+    @FindBy(xpath = "//div[@data-id=%s]/descendant::span[%s]")
+    protected Button middleDropdownBtn;
+
+    public CataloguePage() {
+        super();
+    }
+
+    @Override
+    public CataloguePage openPage() {
+        driver.navigate().to(props.getKeyProperty(Type.CATALOGUE_URL));
+        logger.debug("Navigation to the catalogueURL...");
+        return this;
+    }
+
+    @Override
+    public CataloguePage isPageOpened() {
```

Comment:
method is called 'isPageOpened', i would expect it to return boolean, try - if it's opened, false if not
please rename method to 'waitForPageOpened'

Reply:
renamed